lib/resources/SubscriptionSchedules.js

```diff
@@ -6,14 +6,7 @@ const stripeMethod = StripeResource.method;
 module.exports = StripeResource.extend({
   path: 'subscription_schedules',
 
-  includeBasic: [
-    'create',
-    'list',
-    'retrieve',
-    'update',
-    'setMetadata',
-    'getMetadata',
-  ],
+  includeBasic: ['create', 'list', 'retrieve', 'update'],
```

Comment:
Yeah, fairly sure this is what we want!

Reply:
I think in this case and in the `listSourceTransactions` case, what you see in x-stripe fields is java-specific, not canonically true.

Reply:
The `listRevisions` method in this file is listed in OpenAPI as just `revisions` (I can user overrides to keep it as `listRevisions`, just making sure this is what we want).